### PR TITLE
add tintColor and barTintColor to TabBar

### PR DESCRIPTION
**Pull request by cmcewen**

### CODE REVIEW

File: Libraries/Components/TabBarIOS/TabBarIOS.ios.js [hunk 1]
```diff
@@ -44,7 +55,10 @@ var styles = StyleSheet.create({
 });
 
 var config = {
-  validAttributes: ReactIOSViewAttributes.UIView,
+  validAttributes: merge(ReactIOSViewAttributes.UIView, {
```
Comment (sahrens) [inline Libraries/Components/TabBarIOS/TabBarIOS.ios.js]: This is going to conflict with https://github.com/sahrens/react-native/commit/58a550fa0625f31108e06f32ebb2958e2ffefcf0#diff-27f88ff7e26d700f93b6d9791f3d408d

Can you rebase when that lands?

**Comment by tsunammis** [comment]
I'm looking forward to use this feature

**Comment by cmcewen** [comment]
Didn't see that commit landed - I was watching the wrong PR. Should be good to go now @sahrens

**Comment by tsunammis** [comment]
@cmcewen can you retry the build on Travis ? maybe with the react-native 0.4.1, the issue will be resolved ;-)

**Comment by tsunammis** [comment]
@cmcewen Do you know how fix the failed test ?

**Comment by nicklockwood** [comment]
Sorry this has gone unattended for a while. We're still catching up on the backlog.

This is basically fine aside from a few stylistic issues. If you can fix those, and rebase against the latest version, we can get this merged.

**Comment by cmcewen** [comment]
No worries! I know there's a lot to go through.

**Comment by tsunammis** [comment]
@cmatheson Can I help you on anything ?

**Comment by sallar** [comment]
Can someone please complete this feature? I really need it :-)

**Comment by tsunammis** [comment]
Hello @cmcewen @sallar @nicklockwood 

I rebased @cmcewen work from the master (0.4.4) and by now all the tests pass -> (https://github.com/facebook/react-native/pull/1337)[https://github.com/facebook/react-native/pull/1337]

@cmcewen can you rebase with the master ? else @nicklockwood you can directly merge my PR, it is up to date

@cmcewen I not updated your work and neither your "commits", just rebased ;-)

**Comment by sallar** [comment]
@tsunammis Thanks! I was starting to do it myself, Since I want to use the native tabbar in a project of mine. But will wait for this PR to be merged.

**Comment by tsunammis** [comment]
@sallar Like me :)

**Pull request closed without merging**